File: Day330/DestinationCity.java
class Solution {
    public String destCity(List<List<String>> paths) {
        Map<String, String> map =new HashMap<>();
        for (int i=0; i<paths.size(); i++) {
            map.put(paths.get(i).get(0), paths.get(i).get(1));
        }
        String temp = paths.get(0).get(0);
        while (map.containsKey(temp)) {
            temp = map.get(temp);
        }
        return temp;
    }
}

File: Day330/DestinationCity.go
func destCity(paths [][]string) string {
    m := make(map[string]int)
    for _,v := range paths {
        m[v[0]]++
        m[v[1]]++
    }
    for _,v := range paths {
        if m[v[1]] == 1 {
            return v[1]
        }
    }
    return ""
}